CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Use [gitmoji](https://gitmoji.dev/) to identify your changes.
 
 ### ✨ Added <!--Make sure to add a link to the PR and issues related to your change-->
 - Added sensors name in sensors icons [#473](https://github.com/Metroscope-dev/metroscope-modeling-library/pull/473)
+- Added 4 new units: Volume, Surface tension, Dynamic viscosity and Thermal conductivity [#474](https://github.com/Metroscope-dev/metroscope-modeling-library/pull/474)
```

Comment:
```suggestion
- Added units for volume, surface tension, dynamic viscosity and thermal conductivity [#474](https://github.com/Metroscope-dev/metroscope-modeling-library/pull/474)
```

Reply:
It's better to have the exactly same changelog and PR title (I updated the title)